CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 1.10.0
+Release built: _not released yet_
+
+### API Changes
+- Two new filters to `/stream/transactions` endpoint.
+  - `transaction_status_filter` - Allows filtering by transaction status (`Successful`,`Failed`, `All`). Defaults to `All`.
+  - `balance_change_resources_filter` - Allows specifying an array of resources. If specified, the response will contain transactions which included non-fee related balance changes for all resources.
```

Comment:
```suggestion
- New filters are supported on the `/stream/transactions` endpoint:
  - `transaction_status_filter` - Allows filtering by the transaction commit status (`Success`,`Failure`, `All`). Defaults to `All`.
  - `balance_change_resources_filter` - Allows filtering to transactions which included non-fee related balance changes for all provided resources. Defaults to `[]`. We recommend integrators use this instead of the `manifest_resources_filter` in most instances.